### ISSUE TITLE
Amendments and Origins Classes Compat

### ISSUE THREAD

**Issue by PrinceFeggari** [MehVahdJukaar/amendments#57]
So I found an exploit that could cause problems on some SMPs or Modpacks when you use Origins Classes and Amendments together. Combined with the Clerics ability to double a potions length and the ability to mix potions you can make a potion able to last thousands of hours on end. I recommend making it blacklist the ability to mix potions that have been extended by the Clerics powers
![17k pots](https://github.com/MehVahdJukaar/amendments/assets/142614980/aeb0983a-c89e-44a9-aeb1-2875b5312599)

**Comment by PrinceFeggari** [comment]
> Are you sure you are using the latest version? Mixed pot length should be the average of the ones it was made out of

Yes it was going by the length the potion was. But the Origins Classes mod makes it so that you are able to double potion length making the mixed potion the same length but it counts as a non extended potion meaning it is able to be doubled again. I recommend preventing the ability to mix doubled potions

**Comment by MehVahdJukaar** [comment]
what tag do they add to the item? the one that says if its extended or not?

**Comment by PrinceFeggari** [comment]
This is the data used to mark the potion as extended: {IsExtendedByCleric: 1b}

**Comment by MehVahdJukaar** [comment]
Should be fixed. Extended potions will keep their tag. Might have the side effect to not allow mixing ones that are not extended with ones that are

**Comment by PrinceFeggari** [comment]
Is the update out? I tested it and it seemed to have the same problem still

**Comment by MehVahdJukaar** [comment]
does it wor in newer versions?
